src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -114,84 +101,86 @@ public bool ContentSet
         /// <summary>
         /// Generates the <see cref="TagHelperOutput"/>'s start tag.
         /// </summary>
-        /// <returns><c>string.Empty</c> if <see cref="TagName"/> is <c>null</c> or whitespace. Otherwise, the
+        /// <returns><c>Null</c> if <see cref="TagName"/> is <c>null</c> or whitespace. Otherwise, the
         /// <see cref="string"/> representation of the <see cref="TagHelperOutput"/>'s start tag.</returns>
         public string GenerateStartTag()
```

Comment:
should all of the `Generate...` methods return `ITextWriterCopyable`?  start tags for example are composed of lots of little `string`s we don't need to build into one `string`.
FYI #314 changes this method to use a `TextWriter` internally but it still returns a `string`.